packages/modular-scripts/src/__tests__/utils/createEsbuildBrowserslistTarget.test.ts

```diff
@@ -51,7 +51,7 @@ describe('createEsbuildBrowserslistTarget', () => {
     );
 
     const result = createEsbuildBrowserslistTarget(process.cwd());
-    expect(result).toEqual(['chrome97', 'firefox96', 'safari15.2']);
+    expect(result).toEqual(['chrome99', 'firefox97', 'safari15.2']);
```

Comment:
Can we save the list of unsupported browsers somewhere in the app? It'll help us ensure we are consistent with which browsers we can filtering out and also it'll prevent these hard coded tests from breaking if we miss a line when the list needs to be updated?

Reply:
Yeah - these tests are separate to any web pack piece of this. It just shows that our current version of browsers list is out of date and it's been updated as part of this PR. We could tackle this part of the changeset by updating outside of this context.